prov/efa/src/rxr/rxr_pkt_type_misc.c

```diff
@@ -643,6 +643,20 @@ void rxr_pkt_handle_receipt_recv(struct rxr_ep *ep,
 	if (!tx_entry) {
 		FI_WARN(&rxr_prov, FI_LOG_CQ,
 			"Failed to retrive the tx_entry when hadling receipt packet.\n");
+	}
+
+	tx_entry->receipt_received = true;
+	if (tx_entry->total_len == tx_entry->bytes_acked &&
+	    tx_entry->total_len != 0) {
```

Comment:
why would we need ```tx_entry->total_len !=0```?

Reply:
Unnecessary condition removed. Resolving.